kythe/cxx/common/kzip_writer.cc

```diff
@@ -52,8 +55,16 @@ Status WriteTextFile(zip_t* archive, const std::string& path,
                      absl::string_view content) {
   if (auto source =
           zip_source_buffer(archive, content.data(), content.size(), 0)) {
-    if (zip_file_add(archive, path.c_str(), source, ZIP_FL_ENC_UTF_8) >= 0) {
-      return OkStatus();
+    if (auto idx =
+            zip_file_add(archive, path.c_str(), source, ZIP_FL_ENC_UTF_8);
+        idx >= 0) {
```

Comment:
if-init is a C++17 feature and we're using C++14 due to some issues with the Bazel runtime build environment.

Even if this works, we should restrict to the specified version of C++ for now.

Reply:
done